cmd/manager/main.go

```diff
@@ -119,6 +119,7 @@ func main() {
 	}
 
 	// Checking if required files exist before starting the operator
+	// Note that the metrics wellknownlocations.WindowsExporterPath is (temporarily) not a required file
```

Comment:
@gmarkley-VI and I have been discussing how to get this out to the community first and have some burn time there. Thinking more about it, my initial plan of just controlling that using the Dockerfile seems a bit weird now. So we most probably will go with having a separate branch `release-4.6-community` to track community release. At that point we can cherry-pick features between that and `release-4.6`. Given that I suggest adding the file back here. Sorry about the confusion.